src/Controls/src/Core/HandlerImpl/NavigationPage/NavigationPage.Legacy.cs

```diff
@@ -15,8 +15,7 @@ public partial class NavigationPage : INavigationPageController
 	{
 		internal async Task<Page> PopAsyncInner(
 			bool animated,
-			bool fast,
-			bool requestedFromHandler)
```

Comment:
All of the code was just passing false to `requestedFromHandler` so I removed it.